monitored-httpclient/src/test/java/com/github/nhenneaux/resilienthttpclient/monitoredclientpool/ResilientClientTest.java

```diff
@@ -40,6 +41,7 @@ void send() throws IOException, InterruptedException {
         when(hostAddress.getHostAddress()).thenReturn("10.1.1.1");
 
         final SingleIpHttpClient ipHttpClient = spy(new SingleIpHttpClient(httpClient, hostAddress, new ServerConfiguration(UUID.randomUUID().toString())));
+        doAnswer(invocation -> {return null;}).when(ipHttpClient).checkHealthStatus();
```

Comment:
What about using `org.mockito.Mockito#doNothing`

Reply:
Yes it's a better option, would like me to modify it ?

Reply:
it's not blocker for sure ;-) it's easier to read so I would say yes